src/components/landing/Landing.tsx

```diff
@@ -1,30 +1,79 @@
-import React, {useState} from 'react';
+import React from 'react';
 import { Link } from 'react-router-dom';
-import './Landing.css';
-import terminal from './landing-images/terminal.gif';
+import styled from 'styled-components';
 
 const Landing: React.FC = () => {
-    const [display, setDisplay] = useState('block');
+    const Landing = styled.div`
+        height: 100%;
+        overflow-x: hidden;
+    `
+    
+    const Title = styled.div`
+        position: relative;
+        font-size: 40px;
+        width: 16em;
+        height: 25%;
+        margin: 0 auto;
+        border-right: 2px solid rgba(255, 255, 255, 0);
+        overflow: hidden;
+        transform: translateY(-50%);
+        font-weight: 400;
+        animation: typewriter 1s steps(9) 1s 1 normal both,
+        blinkTextCursor 1s steps(9) infinite normal;
 
-    const startTutorial = () => {
-        setDisplay('none');
-    }
+        @keyframes typewriter {
+            from {
+                width: 0;
+            }
+            to {
+                width: 190px;
+            }
+        }
+        
+        @keyframes blinkTextCursor {
+            0% {
+                border-right-color: rgba(0, 0, 0, 1);
+            }
+            25% {
+                border-right-color: rgba(0, 0, 0, 1);
+            }
+            50% {
+                border-right-color: rgba(0, 0, 0, 1);
+            }
+            75% {
+                border-right-color: transparent;
+            }
+            100% {
+                border-right-color: transparent;
+            }
+        }
+    `
+
+    const Jumbotron = styled.div`
+        height: 350px;
+
+        @media screen and (max-width: 575px) {
+            .jumbotron {
+                padding-top: 60px;
+            }
+        }
+    `
 
     return (
-        <div className="Landing" style={{display: display}}>
+        <Landing className="Landing">
             <div className="container h-100 text-center">
                 <div className="row align-items-center h-100">
-                    <div className="col-lg-6 col-md-8 col-10 offset-lg-3 offset-md-2 offset-1 jumbotron text-center mt-5 mt-xl-0">
-                        <h1 className="title anim-typewriter">
+                    <Jumbotron className="col-lg-6 col-md-8 col-10 offset-lg-3 offset-md-2 offset-1 jumbotron text-center mt-5 mt-xl-0">
+                        <Title className="anim-typewriter">
                             Propguard
-                        </h1>
+                        </Title>
                         <small>A webapp that teaches people how to use Dronekit and Python to start and launch the drones</small> <br></br>
```

Comment:
So visually, I think this text is a little too small. From a development standpoint, I'd prefer if you use CSS to control the font size because the <small> tag allows you to shrink fonts to sizes even tinier than what the font supports.

The reason is that while that <small> a neat feature, doing things like that in HTML (ex: using <b> for bolding fonts) may result in the browser applying font styles that don't come from the font itself. HTML will bold fonts with <b> even if the font doesn't have a bolded version.

This means there will be an inconsistency in what think you think the bolded font should look like and what the font actually is in certain cases. So while it's a _very_ minor issue, it's something to think about. I know that designers in real life can be fairly picky so things like these are very appreciated.